src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -57,21 +57,94 @@ uct_cuda_ipc_mem_add_reg(void *addr, uct_cuda_ipc_memh_t *memh,
 {
     uct_cuda_ipc_lkey_t *key;
     ucs_status_t status;
+#if HAVE_CUDA_FABRIC
+    ucs_log_level_t log_level             = UCS_LOG_LEVEL_ERROR;
```

Comment:
Please remove the variable and replace `UCT_CUDADRV_FUNC(..., log_level);` by `UCT_CUDADRV_FUNC_LOG_ERR(...)`